### DISCUSSION TITLE
Home Assistant OS 11.5.rc3

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3128] [Announcements]
Changes since 11.5.rc2

## Raspberry Pi

* Add optional rpi-rf-mod dtoverlay block for RPi5 (#3080) @jens-maus

## Hardkernel ODROID

* Update ATF and DDR blob for Odroid M1, change 1056->1560 MHz RAM timing (#3125) @sairon
* Disable Ethernet in U-Boot if ODROID-M1 (#3122) @agners

## ASUS Tinker

* Disable MMC environment on ASUS Tinkerboard (#3124) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.5.rc3'>Home Assistant OS 11.5.rc3</a>.</em>

**Comment by AllardKatan** [comment]
After the RC1 crashed my Virtual Machine I reverted to 11.4. Upgraded to RC3 from there and all is fine and dandy. Thanks!

**Comment by codyc1515** [comment]
Do the rc versions of HAOS use the latest beta frontend & core too (or just stable)? I am wanting to confirm some changes in the restore process that are only available in the latest frontend.

**Reply by sairon** [reply]
A fresh OS install always uses the versions from the [stable](https://github.com/home-assistant/version/blob/master/stable.json) channel - some (like Supervisor) are bundled, some are downloaded on the first boot. However, if you upgrade the system, it still uses current versions from the channel that is configured in your Supervisor options, and these are untouched on OS update.
